### PR TITLE
[MV-400] Add more unit tests

### PR DESCRIPTION
**Pull request by skyman503**

### CODE REVIEW

File: example/__tests__/videoroomContext.test.ts [hunk 1]
```diff
@@ -108,4 +129,68 @@ describe('Videoroom context', () => {
...
+      result.current.setRoomName('test room     ');
+      result.current.setUsername('test user   ');
+    });
+    await act(async () => {
+      await result.current.connectAndJoinRoom();
+    });
+
+    expect(result.current.roomName).toBe('test room');
+    expect(result.current.username).toBe('test user');
+    expect(setExtra).toBeCalledTimes(2);
```
Comment (graszka22) [inline example/__tests__/videoroomContext.test.ts:148]: can we check if we set correct values here?

File: example/__tests__/videoroomContext.test.ts [hunk 2]
```diff
@@ -108,4 +129,68 @@ describe('Videoroom context', () => {
...
+      wrapper: VideoroomContextProvider,
+    });
+
+    await act(async () => {
+      await result.current.connectAndJoinRoom();
+      await result.current.disconnect();
+    });
+
+    expect(disconnectCallback).toBeCalledTimes(1);
+    expect(result.current.videoroomState).toBe('AfterMeeting');
```
Comment (graszka22) [inline example/__tests__/videoroomContext.test.ts:165]: I'd add a test for rejoining, it's a bit tricky. Also a test for going back to the main screen (the name and room name are reset).

**Review by graszka22 — COMMENTED** [review]
It's ok, two suggestions

File: example/__tests__/videoroomContext.test.ts (hunk 1 above)
Comment (skyman503) [reply example/__tests__/videoroomContext.test.ts:149]: Done.

File: example/__tests__/videoroomContext.test.ts (hunk 2 above)
Comment (skyman503) [reply example/__tests__/videoroomContext.test.ts:168]: Added rejoining scenario. 
There was already a test checking for going back to the main screen behaviour. I updated it to call `disconnect` before leaving. I think that's enough given that those are supposed to be unit tests only.

**Review by graszka22 — APPROVED** [review]

**Pull request merged**